### PR TITLE
fix(#1289): allow dropdown and popover to be relatively positioned

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

**Comment by Radha-Alberta** [comment]
- [ ] Angular is Fixed, Issue still exists in react.

image

**Review by ArakTaiRoth — DISMISSED** [review]

File: libs/web-components/src/components/popover/Popover.svelte [hunk 1]
```diff
@@ -57,6 +59,7 @@
   $: _padded = toBoolean(padded);
   $: _open = toBoolean(open);
   $: _disabled = toBoolean(disabled);
+  $: _relative = toBoolean(relative);
```
Comment (vanessatran-ddi) [inline libs/web-components/src/components/popover/Popover.svelte:62]: Missing "relative" property in html-data.json

File: libs/web-components/src/components/dropdown/Dropdown.svelte [hunk 2]
```diff
@@ -26,6 +26,7 @@
   export let error: string = "false";
   export let multiselect: string = "false";
   export let native: string = "false";
+  export let relative: string = "false";
```
Comment (vanessatran-ddi) [inline libs/web-components/src/components/dropdown/Dropdown.svelte:29]: Missing "relative" in html-data.json

**Review by vanessatran-ddi — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**